lobster/tools/codebeamer/codebeamer.py

```diff
@@ -210,17 +253,50 @@ def import_tagged(mh, cb_config, items_to_import):
     return rv
 
 
+def ensure_array_of_strings(instance):
+    if isinstance(instance, list) and all(isinstance(item, str) for item in instance):
+        return instance
+    else:
+        return [str(instance)]
+
+
+def parse_cb_config_file(file_name):
+    assert isinstance(file_name, str)
+    assert os.path.isfile(file_name)
+
+    with open(file_name, "r") as file:
+        data = json.loads(file.read())
+
+        provided_config_keys = set(data.keys())
+        supported_references = set(SUPPORTED_REFERENCES)
+
+        if not provided_config_keys.issubset(supported_references):
+            raise Exception("The provided references are not supported! "
+                            "supported referenes: '%s'" % ', '.join(SUPPORTED_REFERENCES))
+
+        json_config = {}
+        for key, value in data.items():
+            json_config[key] = ensure_array_of_strings(value)
```

Comment:
Why is the call to `ensure_array_of_strings` needed? Under which circumstances is `value` a list, and when is it not a list? And why does it have to be a list?

Reply:
If the user uses more than one fieldname for one reference.
Right now our code works with these three variant of config files:

{
"refs" : **["derived from", "derived from2", "derived from3", "derived from4"]**
}
 or
{
"refs" : **"derived from"**
}
 or
{
"refs" : **["derived from"]**
}

Reply:
I used 'ensure_array_of_strings' to ensure that the field names are always a list, because I want to work with **_one_** data type in the rest of the code.